library/stats.py

```diff
@@ -0,0 +1,232 @@
+import GPUtil
+import psutil
+
+import library.config as config
+import library.lcd_comm as lcd
+
+CONFIG_DATA = config.CONFIG_DATA
+
+
+class CPU:
+    @staticmethod
+    def percentage():
+        cpu_percentage = psutil.cpu_percent(interval=CONFIG_DATA['STATS']['CPU']['PERCENTAGE'].get("INTERVAL", None))
+        # print(f"CPU Percentage: {cpu_percentage}")
+
+        if CONFIG_DATA['STATS']['CPU']['PERCENTAGE']['TEXT'].get("SHOW", False):
+            lcd.DisplayText(
+                ser=config.lcd_comm,
+                text=str(int(cpu_percentage)).zfill(3),
+                x=CONFIG_DATA['STATS']['CPU']['PERCENTAGE']['TEXT'].get("X", 0),
+                y=CONFIG_DATA['STATS']['CPU']['PERCENTAGE']['TEXT'].get("Y", 0),
+                font=CONFIG_DATA['STATS']['CPU']['PERCENTAGE']['TEXT'].get("FONT", "roboto/Roboto-Regular.ttf"),
+                font_size=CONFIG_DATA['STATS']['CPU']['PERCENTAGE']['TEXT'].get("FONT_SIZE", 10),
+                font_color=CONFIG_DATA['STATS']['CPU']['PERCENTAGE']['TEXT'].get("FONT_COLOR", (0, 0, 0)),
+                background_color=CONFIG_DATA['STATS']['CPU']['PERCENTAGE']['TEXT'].get("BACKGROUND_COLOR", (255, 255, 255)),
+                background_image=CONFIG_DATA['STATS']['CPU']['PERCENTAGE']['TEXT'].get("BACKGROUND_IMAGE", None)
+            )
+
+        if CONFIG_DATA['STATS']['CPU']['PERCENTAGE']['GRAPH'].get("SHOW", False):
+            lcd.DisplayProgressBar(
+                ser=config.lcd_comm,
+                x=CONFIG_DATA['STATS']['CPU']['PERCENTAGE']['GRAPH'].get("X", 0),
+                y=CONFIG_DATA['STATS']['CPU']['PERCENTAGE']['GRAPH'].get("Y", 0),
+                width=CONFIG_DATA['STATS']['CPU']['PERCENTAGE']['GRAPH'].get("WIDTH", 0),
+                height=CONFIG_DATA['STATS']['CPU']['PERCENTAGE']['GRAPH'].get("HEIGHT", 0),
+                value=int(cpu_percentage),
+                min_value=CONFIG_DATA['STATS']['CPU']['PERCENTAGE']['GRAPH'].get("MIN_VALUE", 0),
+                max_value=CONFIG_DATA['STATS']['CPU']['PERCENTAGE']['GRAPH'].get("MAX_VALUE", 100),
+                bar_color=CONFIG_DATA['STATS']['CPU']['PERCENTAGE']['GRAPH'].get("BAR_COLOR", (0, 0, 0)),
+                bar_outline=CONFIG_DATA['STATS']['CPU']['PERCENTAGE']['GRAPH'].get("BAR_OUTLINE", False),
+                background_image=CONFIG_DATA['STATS']['CPU']['PERCENTAGE']['GRAPH'].get("BACKGROUND_IMAGE", None)
```

Comment:
background_color attribute is missing (same for other progress bars in this file)